CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## Next Release
+
+- Fix pagination parameters for `GetNextPageOfChildren` function
```

Comment:
```suggestion
- Fix pagination parameters for `GetNextPageOfChildren` function for User service
```

Reply:
Lol, Copilot recommended this as well, and I said no. But on an actual human recommendation, sure.

Reply:
Oh wow, Copilot can detect which service you change now.